### PR TITLE
Add example callback for fiat to crypto flow

### PR DESCRIPTION
**Pull request by badborg**
Ticket reference:
https://ortechnologies.atlassian.net/browse/EEP-39

### CODE REVIEW

File: source/includes/_fiat_to_crypto_flow.md [hunk 1]
```diff
@@ -218,21 +218,50 @@ So the 2 potential cases are:
...
+-d '{
+    "invoice_id": "10e35c16-3ff8-4238-b5b5-34b4da9b4115",
+    "timestamp_utc": "2020-01-01 12:45:00.000000+00",
+    "status": {
+        "completed": {}
+    }
+}'
+```
+
+> Request JSON Body
```
Comment (ivan-mudrak) [inline source/includes/_fiat_to_crypto_flow.md:242]: Actually, it does not reflect the actual situation. In reality, we will send a typical fiat Ingres callback not this one, it was probably wrongly copied from s2s egress flow which we were finalizing at that time also. I will update API docs on separate PR later.

File: source/includes/_fiat_to_crypto_flow.md [hunk 2]
```diff
@@ -218,21 +218,50 @@ So the 2 potential cases are:
...
+| invoice_id    | String | Internal **ON/RAMP**'s invoice identifier.       |
+| timestamp_utc | Date   | Timestamp when transaction has been completed.          |
+| status        | Status | Contains "completed" field when transaction is completed. |
+
+
+### Expected Response
 
-The `status` field can have the following values:
-    - `completed`: The payment has been succesfully completed.
+| Status |
```
Comment (yowlu) [inline source/includes/_fiat_to_crypto_flow.md:265]: This is where we will add the tx_hash if available?

Comment (ivan-mudrak) [reply source/includes/_fiat_to_crypto_flow.md:265]: If we implement 2 callbacks then yes.
The simplest solution is to re-use fiat callback and include it under crypto_info, like this:
```
{
  "reference": "ac400127-93a9-4b9c-9612-c23a3c078933",
  "invoice_id": "62b570e8-9723-4287-a5a8-e825c2ffced2",
  "crypto_info": {
    "tx_hash": "220c540c074ff2744d28f7a37a89a55c8a080590a68da6c2a9d3460ec01b3385"
  }
}
```

**Comment by ivan-mudrak** [comment]
> @ivan-mudrak Do we support signature version like fiat ingress callback?

Actually, we do, but I'm not sure it is needed here. So it will work by default.

File: source/includes/_fiat_to_crypto_flow.md [hunk 3]
```diff
@@ -216,13 +214,29 @@ So the 2 potential cases are:
 - Unverified traffic
 
   - You can either not send the `kyc_verified` parameter or send it with a `null` value.
-  
 
-### Completion Callback
+## Notification of successful transaction
 
-Once transfer is completed, we will do a completion callback to the merchant. Merchant should answer the callback with an http 200 status code.
+A URL provided by the merchant in a previous request (see `callback_url` and `cancel_callback` in [Create fiat to crypto invoice](#create-fiat-to-crypto-invoice)) will be used to send a notification request once the transaction result has been confirmed.  If the transaction is successful, `callback_url` will be used, while `cancel_callback` will be used when the transaction cannot be completed.
```
Comment (ivan-mudrak) [inline source/includes/_fiat_to_crypto_flow.md:220]: Why `payment_ack_url` is changed to `callback_url`?

**Pull request merged**